web/docs/understand/language/frontends/sigma.md

```diff
@@ -2,7 +2,7 @@
 
 The Sigma query frontend makes it possible to execute [Sigma
 rules](https://github.com/SigmaHQ/sigma) in VAST. This means you can
-provide a Sigma rule instead of a VASTQL expression when querying data. For
+provide a Sigma rule instead of a VAST expression when querying data. For
```

Comment:
You could link this to the Expressions page.

Reply:
I added a link in https://github.com/tenzir/vast/pull/2959/commits/1a2ee51cc4e2ae9a3733fae548a8dd5f2b698e22